tools/assembly-store-reader/AssemblyStoreManifestReader.cs

```diff
@@ -7,7 +7,7 @@ namespace Xamarin.Android.AssemblyStore
 {
 	class AssemblyStoreManifestReader
 	{
-		static readonly char[] fieldSplit = new char[] { ' ' };
+		static readonly char[] fieldSplit = new char[] { '\t' };
```

Comment:
@grendello so I'm trying to get this PR up and running again. 

It has to do with an assembly having a space in its name. I think I made this change to use tabs as a delimiter in the assembly store rather than spaces. The old assemblystore this was using has been deleted it seems. 

Do we even need this anymore? Should this particular change be backed out (I think we are are v2 of the assembly store now?)

Reply:
The manifest remained the same, though. IIRC, the problem was never with the actual store but just with the manifest?